ci/steps/publish-npm.sh

```diff
@@ -51,6 +51,15 @@ main() {
     exit 1
   fi
 
+  # TODO@jsjoeio
+  # Check that we're using at least v7 of npm CLI
+  if ! command -v npm &> /dev/null; then
```

Comment:
Hmm could be tough since we would need to do semver checks.  It might be better to just run the `npm config set` command but on failure echo a message saying something like "failed to set package.json name, are you on v7 or higher?"

Reply:
That's a great alternative!